app/routes/events/list.js

```diff
@@ -88,6 +88,8 @@ export default class extends Route.extend(EmberTableRouteMixin) {
       'page[size]'   : params.per_page || 10,
       'page[number]' : params.page || 4
     };
+    params.sort_by = 'starts-at';
+    params.sort_dir = 'DSC';
```

Comment:
This will override the sort parameter, so I won't be able to sort by name, etc

Reply:
ok, i got it. can you plaese suggest what change do i need to make here?

Reply:
Override only if the parameter doesn't exist

Reply:
i have changed the code how can i add the commit to existing pr?